src/providers/okta.ts

```diff
@@ -0,0 +1,102 @@
+import { OAuth2Client } from "oslo/oauth2";
+import type { OAuth2ProviderWithPKCE } from "../index.js";
+import { TimeSpan, createDate } from "oslo";
+
+export class Okta implements OAuth2ProviderWithPKCE {
+	private client: OAuth2Client;
+	private clientSecret: string;
+
+	constructor(
+		oktaDomain: string,
```

Comment:
The domain should include the protocol (`https://`) to be consistent with other providers